src/plugins/event-plugins/__tests__/XhrPlugin.test.ts

```diff
@@ -120,7 +120,7 @@ describe('XhrPlugin tests', () => {
                 }
             ]
         });
-        expect(plugin.cacheSize).toEqual(0);
+        expect((plugin as any).xhrMap.size).toEqual(0);
```

Comment:
issue: Each unit test should verify exactly one property of the program. In this case, we want to verify that `xhrMap` is empty after the XHR event was recorded. To accomplish this, we can a new unit test which verifies that property -- something like *"when there are no more active xhrs then the cache is empty"*.

Are there any other cases that we should test? Do we have reasonable path coverage (e.g., do we cover successful and error cases)?

Reply:
> we can add a new unit test which verifies that property.

Created tests for Xhr success, error, abort, and timeout

> are there any other cases that we should test?

Maybe the DOMEventPlugin, I'd have to read through everything first.